### PR TITLE
[Issue# 1678]: Added Wiki link to Process Page on Static Site

### PR DESCRIPTION
**Pull request by btabaska**
## Summary
Fixes #{1678}

## Additional information
Screenshot 2024-04-15 at 9 56 01 AM

### CODE REVIEW

**Review by andycochran — DISMISSED** [review]
LGTM — tho we need content approval before merging (deploying?) this change

**Comment by andycochran** [comment]
@lucasmbrown-usds / @itsemilyianacone — Adding you as reviewers. Please let us know if more involved content review is necessary.

**Comment by btabaska** [comment]
> LGTM — tho we need content approval before merging (deploying?) this change

We got approval in Slack, we should be good to merge.

**Review by andycochran — APPROVED** [review]

**Pull request merged**
